src/Compilers/CSharp/Portable/Errors/MessageID.cs

```diff
@@ -252,12 +252,11 @@ internal enum MessageID
         IDS_FeatureUtf8StringLiterals = MessageBase + 12822,
 
         IDS_FeatureUnsignedRightShift = MessageBase + 12823,
-        IDS_FeatureExtendedNameofScope = MessageBase + 12824,
```

Comment:
Is it fine to reuse this ID? I guess it is since the compiler only uses the IDE internally, it's not persisted in IDE settings, etc.

Reply:
Yup